src/pages/Portfolio/LendingStats/LendingStatsPage.vue

```diff
@@ -136,23 +134,29 @@ export default {
 		query: lendingStatsQuery,
 		preFetch: true,
 		result({ data }) {
-			const allCountries = _sortBy(_map(_get(data, 'lend.countryFacets'), 'country'), 'name');
-			this.countriesLentTo = _sortBy(_get(data, 'my.lendingStats.countriesLentTo'), 'name');
+			const countriesWhereKivaDoesntWork = data?.general?.excluded_countries?.value ?? '';
```

Comment:
Should the fallback be `[]`?

Reply:
No settings always come back as strings. This is actually just a string of comma separated values. Down below. `.includes` works on strings anyway